component/Grid/grid.html

```diff
@@ -0,0 +1,198 @@
+					></a>
+				</li>
+				<li class="list-item-inline">
+					<a class="link" href="https://www.linkedin.com/in/unv/">
+						<i class="fa fa-linkedin-square social-icons"></i
+					></a>
+				</li>
+				<li class="list-item-inline">
+					<a class="link" href="https://twitter.com/UNVishwakarma">
+						<i class="fa fa-twitter social-icons"></i
+					></a>
+				</li>
+			</ul>
+		</nav>
+		<main class="main-container">
+			<div class="side-bar">
+				<h2 class="sidebar-heading">Components</h2>
+				<a class="sidebar-subheading" href="/component/Alert/alert.html"
+					>Alert</a
+				>
+				<a class="sidebar-subheading" href="/component/Avatar/avatar.html"
+					>Avatar</a
+				>
+				<a class="sidebar-subheading" href="/component/Badges/badge.html"
+					>Badges</a
+				>
+				<a class="sidebar-subheading" href="/component/Buttons/button.html"
+					>Buttons</a
+				>
+				<a class="sidebar-subheading" href="/component/Card/card.html">Card</a>
+				<a
+					class="sidebar-subheading sidebar-link-active"
+					href="/component/Grid/grid.html"
+				>
+					Grid
+				</a>
+				<a class="sidebar-subheading" href="/component/Image/Image.html"
+					>Image</a
+				>
+				<a class="sidebar-subheading" href="/component/Input/Input.html"
+					>Input</a
+				>
+
+				<a
+					class="sidebar-subheading"
+					href="/component/Navigation/navigation.html"
+					>Navigation</a
+				>
+				<a class="sidebar-subheading" href="/component/Rating/rating.html"
+					>Rating</a
+				>
+				<a class="sidebar-subheading" href="/component/Toast/toast.html"
+					>Toast</a
+				>
+				<a
+					class="sidebar-subheading"
+					href="/component/Typography/typography.html"
+					>Typography</a
+				>
+			</div>
+			<section class="content-container content-wrapper">
+				<div class="sectn">
+					<div class="content-header h1">Grid</div>
+					<p class="md-text">
+						CSS Grid can be used to structure elements in rows and columns.</p
+					>
+					<h2 class="h2">Two Columns Grid</h2>
+					<div class="component-display-container">
+						<div class="grid col-2">
+							<div>Grid Column 1</div>
+							<div>Grid Column 2</div>
+						</div>
+					</div>
+					<div class="code-snippet-container">
+						<iframe
+							src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=material&wt=boxy&l=auto&width=825&ds=false&dsyoff=20px&dsblur=68px&wc=false&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2509%2509%2509%2509%2509%253Cdiv%2520class%253D%2522component-display-container%2522%253E%250A%2509%2509%2509%2509%2509%2509%253Cdiv%2520class%253D%2522grid%2520col-2%2522%253E%250A%2509%2509%2509%2509%2509%2509%2509%253Cdiv%253EGrid%2520Column%25201%253C%252Fdiv%253E%250A%2509%2509%2509%2509%2509%2509%2509%253Cdiv%253EGrid%2520Column%25202%253C%252Fdiv%253E%250A%2509%2509%2509%2509%2509%2509%253C%252Fdiv%253E%250A%2509%2509%2509%2509%2509%253C%252Fdiv%253E"
+							style="
+								width: 825px;
+								height: 139px;
```

Comment:
Use rems for length units

Reply:
I tried using rems instead of px but since there are various classes in carbon iframe therefore it doesn't work as expected.